corehq/apps/es/es_query.py

```diff
@@ -143,7 +143,7 @@ class ESQuery(object):
 
     def __init__(self, index=None, debug_host=None):
         self.index = index if index is not None else self.index
-        if self.index not in ES_META:
+        if self.index not in ES_META and not self.index.startswith('config_report'):
```

Comment:
should probably pull out this `index.startswith('config_report')` business into a helper function